coinapi/helm-charts
# Updated, expanded and consolidated all information into just one README.

Ok, following my deployment experience, 
I had to expand the readme substantially to address a couple of points, among others:

* Complete and working config example
* Link to API docs i.e. where to find supported exchanges 
* How to verify installation 
* How to connect to composite API 

The last one is became necessary as port-forwarding doesn't work so, I documented a work-around to test if the system works.

Also, for practical reasons one single readme file should be sufficient. 
 
What's still missing is a proper development setup, but I guess that's for another day.
